src/lyapunovs.jl

```diff
@@ -213,20 +214,20 @@ end
 #                            Continuous Lyapunovs                                   #
 #####################################################################################
 function default_rescale(ds)
-  D = dimension(DS)
-  (state2, state1, d0) -> broadcast!(+, state2, state1, d0/sqrt(D))
+  sqD = sqrt(dimension(ds))
+  (state2, state1, d0) -> broadcast!(+, state2, state1, d0/sqD)
 end
 
 function lyapunov(ds::ContinuousDynamicalSystem,
-                  T::Real = 10000.0,
-                  return_convergence::Val{B} = Val{false};
+                  T::Real,
+                  return_convergence::Val{B} = Val{false}();
```

Comment:
Don't need the `()` if you do `return_convergence::Type{Val{B}}` which is better accepted.